papis/tui/utils.py

```diff
@@ -1,6 +1,50 @@
 import re
 from typing import Optional, List, Callable, Any
 
+PAPIS_PYGMENTS_DEFAULT_STYLE = {
```

Comment:
Are these taken from somewhere? Maybe add a little note explaining why it's like this if possible?

Reply:
I just used a copy of the default style and changed it to use ansi colors. I'll make a note of it, thanks!